src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = (props) => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (inputText.title.trim()) {
+      props.addTodoProps(inputText.title);
+      setInputText({
+        title: '',
+      });
+    } else {
+      // eslint-disable-next-line no-alert
+      alert('Please write item');
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit} className="form-container">
+      <input
+        type="text"
+        className="input-text"
+        placeholder="Add todo..."
+        value={inputText.title}
+        name="title"
+        onChange={onChange}
+      />
+      <button type="button" className="input-submit">
+        <FaPlusCircle
+          style={{ color: 'darkcyan', fontSize: '20px', marginTop: '2px' }}
+        />
+      </button>
```

Comment:
### Your [InputTodo.js](https://github.com/DJ-MrJay/react-todo-app/pull/1/files#diff-89941f230a757f229a14e17006ff835825d86cb1f366b2d35eea5e9bf67ea015) File

![image](https://user-images.githubusercontent.com/98043302/192999499-e5b75299-79c1-44a5-89b0-a9e36040437f.png)

[Optional]

-  When I click the button it doesn't work the add only works by the "enter" key so kindly fix it  😊

Reply:
Thank you so much. i will fix it